packages/components/package.json

```diff
@@ -21,6 +21,8 @@
   "devDependencies": {
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
-    "react-test-renderer": "^16.8.4"
+    "react-hooks-testing-library": "^0.3.8",
+    "react-test-renderer": "^16.8.4",
+    "sinon": "^7.3.1"
```

Comment:
I'd rather not add the extra dependencies – I'm not sure if `sinon` adds anything that Jest doesn't already have and for testing hooks, I'd rather see unit tests for components that use the hooks

Reply:
My bad, I'm not familiar with Jest and I should have looked to see if it had spies built in. I do agree with testing components that use the hooks however, I feel that there also should be tests specifically testing the hook. That way it's a bit more explicit to what the hook is expected to do. Also if in the future a test needs to be created to make sure there isn't any regression.

Reply:
The thinking with not testing the hooks directly is that I'd rather test the end API for what the hook does. It's unlikely, but if the implementation changes, I'd rather the unit tests still pass if the API still works, if that makes sense